packages/inventory/src/EntityTableToolbar.js

```diff
@@ -1,61 +1,111 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import { Filter } from './Filter';
-import { TableToolbar, Skeleton, SkeletonSize } from '@redhat-cloud-services/frontend-components';
-import { Split, SplitItem } from '@patternfly/react-core';
-
-const EntityTableToolbar = ({
-    total,
-    page,
-    onRefresh,
-    perPage,
-    filters,
-    hasItems,
-    pathPrefix,
-    apiBase,
-    children,
-    pagination,
-    loaded,
-    ...props
-}) => (
-    <TableToolbar className="ins-c-inventory__table--toolbar">
-        <Split>
-            <SplitItem>
-                <Filter { ...props }
-                    hasItems={ hasItems }
-                    filters={ filters }
-                    pathPrefix={ pathPrefix }
-                    apiBase={ apiBase }
-                    totalItems={ total }
-                />
-            </SplitItem>
-            <SplitItem isFilled>
-                { children }
-            </SplitItem>
-            <SplitItem>
-                {
-                    loaded ?
-                        pagination :
-                        <Skeleton size={ SkeletonSize.lg } />
-                }
-            </SplitItem>
-        </Split>
-    </TableToolbar>
-);
+import { Skeleton, SkeletonSize, PrimaryToolbar } from '@redhat-cloud-services/frontend-components';
+import debounce from 'lodash/debounce';
+
+const TEXT_FILTER = 'hostname_or_id';
+
+class EntityTableToolbar extends Component {
+    state = {
+        textFilter: ''
+    }
+
+    debouncedRefresh = debounce((config) => {
+        this.props.onRefresh && this.props.onRefresh(config);
+    }, 800);
+
+    onSetTextFilter = (value) => {
+        const { page, perPage } = this.props;
+        this.debouncedRefresh({
+            page, perPage, filters: [{
+                value: TEXT_FILTER,
+                filter: value
+            }]
+        });
+        this.setState({
+            textFilter: value
+        });
+    }
+
+    render() {
+        const {
+            total,
+            page,
+            onRefresh,
+            perPage,
+            filters,
+            hasItems,
+            // eslint-disable-next-line no-unused-vars
```

Comment:
we do this more than once, time to throw `/* eslint-disable no-unused-vars */` on the top